### PR TITLE
Adds Western Frontier Pack

### PR DESCRIPTION
**Pull request by Xantholne**
Adds new Western Frontier Pack available in the loadout menu.
Contains several hats, boots, shirts, and short sleeved shirts. Shirts work like accessories so you are not stuck with only one color scheme and can change them to any pair of jeans.
41f5efa14455485331c2dab84c49f8f2
4ad2569ae6d10b40c66e8f3488791054
_Is High Noons _yaya__

🆑 Xantholne
rcsadd: Adds Frontier megapack to the loadout menu with new hats, boots, and clip on accessory shirts
/🆑

### CODE REVIEW

**Comment by Rapidvalj** [comment]
🌈 pink cowboy boots, amazing

**Comment by pinatacolada** [comment]
Its high noon

**Comment by PhantasmicDream** [comment]
You can truly become a space cowboy.

**Comment by Lady-Luck** [comment]
I like it, but can you make the pink less eye-rapey.

**Comment by fludd12** [comment]
Pretty sure that is the point of the pink.

**Comment by Spacemanspark** [comment]
Looks nice.

**Comment by taukausanake** [comment]
Don't these need species sprites too?

File: code/modules/clothing/head/misc.dm [hunk 1]
```diff
@@ -188,6 +188,30 @@
 	item_state = "fedora"
 	desc = "There's a new sheriff in town. Pass the whiskey."
 
+/obj/item/clothing/head/cowboyhat_brown
```
Comment (Fox-McCloud) [inline code/modules/clothing/head/misc.dm]: we already have a brown cowboy hat; it's the one above.

File: code/modules/clothing/head/misc.dm [hunk 2]
```diff
@@ -188,6 +188,30 @@
 	item_state = "fedora"
 	desc = "There's a new sheriff in town. Pass the whiskey."
 
+/obj/item/clothing/head/cowboyhat_brown
+	name = "cowboy hat"
+	icon_state = "cowboyhat_brown"
+	item_state = "cowboyhat_brown"
+	desc = "For the Rancher in us all."
+
+/obj/item/clothing/head/cowboyhat_black
```
Comment (Fox-McCloud) [inline code/modules/clothing/head/misc.dm]: these should all be a subtype of cowboy hat.

File: code/modules/clothing/shoes/miscellaneous.dm [hunk 3]
```diff
@@ -232,3 +232,27 @@
 	desc = "These boots were made for dancing."
 	icon_state = "bsing"
 	put_on_delay = 50
+
+/obj/item/clothing/shoes/cowboyboots
```
Comment (Fox-McCloud) [inline code/modules/clothing/shoes/miscellaneous.dm]: we already have brown cowboy boots; now need for duplicate.

Also, again, all color varients should be a subtype.

File: code/modules/clothing/under/accessories/accessory.dm [hunk 4]
```diff
@@ -377,6 +377,117 @@
...
+	desc = "For the rancher in us all."
+	icon_state = "cowboyshirt_white"
+	item_state = "cowboyshirt_white"
+	item_color = "cowboyshirt_white"
+	species_fit = list("Vox")
+	sprite_sheets = list(
+		"Vox" = 'icons/mob/species/vox/suit.dmi'
+		)
+
+/obj/item/clothing/accessory/cowboyshirt/whites
```
Comment (Fox-McCloud) [inline code/modules/clothing/under/accessories/accessory.dm]: following the subtyping standard you established, above, please.

**Comment by taukausanake** [comment]
So I kind of just noticed something but I doubt it means anything. The original cowboy hat which I'm thinking is in the current line 185 (the one with the fedora item_state) is inside a section that is defined as "stylish bs12 hats" by comments and it is below this hat and in this section where the new hats are being defined. These are not BayStation12 hats at least I don't think so? Then again what are comments good for anyways?

**Comment by Xantholne** [comment]
No they aren't. But it's more appropriate to make a subtype belong the main thing than well, in an entirely new area itself. I don't think Fox would even like that.

**Comment by ZomgPonies** [comment]
Pink cowboy gear? Submissive Steve is happy!

File: code/modules/clothing/head/misc.dm [hunk 5]
```diff
@@ -188,6 +188,30 @@
...
+	name = "dark brown cowboy hat"
+	icon_state = "cowboyhat_darkbrown"
+	item_state = "cowboyhat_darlbrown"
+	desc = "For the Rancher in us all."
+
+/obj/item/clothing/head/cowboyhat/black
+	name = "black cowboy hat"
+	icon_state = "cowboyhat_black"
+	item_state = "cowboyhat_black"
+	desc = "This station aint big enough for the two ah' us."
```
Comment (Birdtalon) [inline code/modules/clothing/head/misc.dm]: Should really be "ain't".

Comment (Birdtalon) [inline code/modules/clothing/head/misc.dm]: Using "ain't" is better form.

File: code/modules/clothing/under/accessories/accessory.dm [hunk 6]
```diff
@@ -377,6 +377,117 @@
...
+	desc = "For a real western look."
+	icon_state = "cowboyshirt"
+	item_state = "cowboyshirt"
+	item_color = "cowboyshirt"
+	species_fit = list("Vox")
+	sprite_sheets = list(
+		"Vox" = 'icons/mob/species/vox/suit.dmi'
+		)
+
+/obj/item/clothing/accessory/cowboyshirt/s
```
Comment (Fox-McCloud) [inline code/modules/clothing/under/accessories/accessory.dm]: `/obj/item/clothing/accessory/cowboyshirt/short_sleeved`

File: code/modules/clothing/under/accessories/accessory.dm (hunk 4 above)
Comment (Fox-McCloud) [inline code/modules/clothing/under/accessories/accessory.dm]: `/obj/item/clothing/accessory/cowboyshirt/white/short_sleeved`

File: code/modules/clothing/under/accessories/accessory.dm [hunk 7]
```diff
@@ -377,6 +377,117 @@
...
+	desc = "For only the manliest of men, or girliest of girls."
+	icon_state = "cowboyshirt_pink"
+	item_state = "cowboyshirt_pink"
+	item_color = "cowboyshirt_pink"
+	species_fit = list("Vox")
+	sprite_sheets = list(
+		"Vox" = 'icons/mob/species/vox/suit.dmi'
+		)
+
+/obj/item/clothing/accessory/cowboyshirt/pinks
```
Comment (Fox-McCloud) [inline code/modules/clothing/under/accessories/accessory.dm]: `/obj/item/clothing/accessory/cowboyshirt/pink/short_sleeved`

File: code/modules/clothing/under/accessories/accessory.dm [hunk 8]
```diff
@@ -377,6 +377,117 @@
...
+	desc = "Now yer a real cowboy."
+	icon_state = "cowboyshirt_navy"
+	item_state = "cowboyshirt_navy"
+	item_color = "cowboyshirt_navy"
+	species_fit = list("Vox")
+	sprite_sheets = list(
+		"Vox" = 'icons/mob/species/vox/suit.dmi'
+		)
+
+/obj/item/clothing/accessory/cowboyshirt/navys
```
Comment (Fox-McCloud) [inline code/modules/clothing/under/accessories/accessory.dm]: `obj/item/clothing/accessory/cowboyshirt/navy/short_sleeved`

File: code/modules/clothing/under/accessories/accessory.dm [hunk 9]
```diff
@@ -377,6 +377,117 @@
...
+	desc = "It's high noon."
+	icon_state = "cowboyshirt_red"
+	item_state = "cowboyshirt_red"
+	item_color = "cowboyshirt_red"
+	species_fit = list("Vox")
+	sprite_sheets = list(
+		"Vox" = 'icons/mob/species/vox/suit.dmi'
+		)
+
+/obj/item/clothing/accessory/cowboyshirt/reds
```
Comment (Fox-McCloud) [inline code/modules/clothing/under/accessories/accessory.dm]: `/obj/item/clothing/accessory/cowboyshirt/red/short_sleeved`

**Comment by Fox-McCloud** [comment]
make sure all your icon states are properly named.

**Comment by Xantholne** [comment]
I'll likely just rename the old one to something like tan; may keep it on the loadout menu then. That way it's not entirely gone nd if people don't like the brown replacement over the original they can use that for their westerner characters

**Pull request merged**